src/main/java/uk/ac/cam/cl/quebec/face/FaceDaemon.java

```diff
@@ -18,24 +18,12 @@
 public class FaceDaemon
 {
     private String mQueueUrl;
-    private String mQueueUser;
-    private String mQueuePass;
 
     private boolean setQueueUrl(String url)
     {
         mQueueUrl = url;
         return true;
     }
-    private boolean setQueueUser(String user)
-    {
-        mQueueUser = user;
-        return true;
-    }
-    private boolean setQueuePass(String password)
-    {
-        mQueuePass = password;
-        return true;
-    }
 
     // Temporary fake queue
     List<Message> tempQueue = new ArrayList<>();
```

Comment:
This should look something along the lines of:
List <Message> messageQueue = makeDummyQueue();